2024/src/pages/index.tsx

```diff
@@ -263,7 +266,7 @@ export default function IndexPage() {
       subTitle: t("callForSponsors"),
       available: site.siteMetadata.sponsorFormUrl,
```

Comment:
The `!sponsorDateOverDue` should be in here, shouldn't it?

Reply:
My thought process here was:

If sponsorFormURL is available, show the sponsor section
If sponsorDeadline has passed, show coming soon
If sponsorDeadline is in the future, show the link to the form

If this is not desirable then let me know.